ddf-test/src/main/scala/io/ddf/test/it/etl/TransformationHandlerBaseSuite.scala

```diff
@@ -204,4 +204,38 @@ trait TransformationHandlerBaseSuite extends BaseSuite with Matchers {
       smithsData.Transform.sort(columns, new util.ArrayList[Boolean]())
     }
   }
+
+  test("test sort inPlace false") {
+    val ddf: DDF = loadSmithsDDF().VIEWS.project("subject, variable, value")
+    val columns = new util.ArrayList[String]()
```

Comment:
Is there a shorter way to specify columns and ascending? May be construct a Scala list and convert to the Java one